polylogue/cli/open_helper.py

```
@@ -44,12 +44,18 @@ def run_open_cli(args: SimpleNamespace, env: CommandEnv) -> None:
     if getattr(args, "print_only", False):
         ui.console.print(str(target))
         return
-    if not open_in_editor(target):
-        editor = get_editor()
-        if editor:
-            ui.console.print(f"[yellow]Could not open with {editor}; path: {target}")
-        else:
-            ui.console.print(f"[yellow]Set $EDITOR to open files automatically. Path: {target}")
+    if target.suffix.lower() == ".html":
+        if open_in_browser(target):
```

Ruff (0.15.9)

[warning] Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)